### PR TITLE
Fixing validation to not force public RDL users to have a volume.

### PR DESCRIPTION
**Pull request by TimSimpsonR**

### CODE REVIEW

File: reddwarf/instance/service.py [hunk 1]
```diff
@@ -317,9 +317,18 @@ def _validate(body):
             vol_enabled = utils.bool_from_string(
                                 config.Config.get('reddwarf_volume_support',
                                                   'True'))
+            must_have_vol = utils.bool_from_string(
```
Comment (rnirmal) [inline reddwarf/instance/service.py:320]: Why are we doing the exact same thing. the vol_enabled above is the same as must_have_vol.

File: reddwarf/instance/service.py [hunk 2]
```diff
@@ -317,9 +317,18 @@ def _validate(body):
...
+                                config.Config.get('reddwarf_must_use_volume',
+                                                  'False'))
             if vol_enabled:
```
Comment (rnirmal) [inline reddwarf/instance/service.py:323]: and we have if vol_enabled here. So not sure that this commit changes anything.

Update: ignore

File: reddwarf/instance/service.py (hunk 1 above)
Comment (TimSimpsonR) [inline reddwarf/instance/service.py:320]: Not at all. vol_enabled means we support optionally having volumes in RDL, while must_have_vol means its required.

File: reddwarf/instance/service.py [hunk 3]
```diff
@@ -317,9 +317,18 @@ def _validate(body):
             vol_enabled = utils.bool_from_string(
                                 config.Config.get('reddwarf_volume_support',
                                                   'True'))
```
Comment (rnirmal) [inline reddwarf/instance/service.py:319]: Lets change the default for the above value as False and update the conf with the same. That way we don't even have to bother with volumes in the external code. The test.conf can have it, but not the default

File: reddwarf/instance/service.py (hunk 1 above)
Comment (rnirmal) [inline reddwarf/instance/service.py:320]: yeah it's all confusing... but I understand... ok

**Pull request merged**